src/components/Input/Toggle/Toggle.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import Toggle from './Toggle';
+
+describe('Toggle', () => {
+  it('renders', () => {
+    render(<Toggle readOnly />);
+  });
+
+  describe('when rendered with id attribute', () => {
```

Comment:
Minor, but a test case that it calls the `onChange` handler with the expected value would be nice.
I believe the expected value would be an event with a target element that has the correct `checked` attribute.

Could easily be a follow-on ticket, if you prefer.